src/Chapter1.hs

```diff
@@ -503,7 +504,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if abs x == min (abs x) (abs y) then x else y
```

Comment:
That's a valid implementation, but I think you can slightly improve it. Note, that you only need to compare `abs x` and `abs y` only once to get the result.